a3p-integration/proposals/z:acceptance/test/stakeBld.test.js

```diff
@@ -84,7 +93,10 @@ test('basic', async t => {
         source: 'continuing',
         previousOffer: 'request-stake',
         invitationMakerName: 'Undelegate',
-        invitationArgs: [VALIDATOR_ADDRESS, { brand: brand.BLD, value: 5n }],
+        invitationArgs: [
+          VALIDATOR_ADDRESS,
+          { brand: brand.BLD, value: BigInt(BLD_DELEGATION / 2) },
```

Comment:
If BLD_DELEGATION is a Number, then this code bakes in an assumption that it must be a multiple of two.  Otherwise, we'll get `RangeError: The number XXX cannot be converted to a BigInt because it is not an integer`.

We can avoid that gotcha by making BLD_DELEGATION a BigInt instead:

```suggestion
          { brand: brand.BLD, value: BLD_DELEGATION / 2n },
```